setup.py

```diff
@@ -21,10 +21,12 @@
     zip_safe=False,
     install_requires=[
         # -*- Extra requirements: -*-
+        "ckanapi"
```

Comment:
@mattboehm Should "ckanapi" be added to the requirements-test.txt as well for local testing?